python/chemiscope/sphinx/static/chemiscope-sphinx.js

```diff
@@ -36,7 +43,7 @@ async function loadChemiscopeSphinx(divId, filePath, visualizerMode = VISUALISER
 
         // Load widget
         const visualiser = getVisualizer(visualizerMode);
-        await visualiser.load(config, dataset);
+        let viz = await visualiser.load(config, dataset, warnings);
```

Comment:
why the `let`?

Reply:
well-spotted, leftover from console.logs I think